spec/features/creating_a_dfid_research_output_spec.rb

```diff
@@ -7,6 +7,7 @@
   let(:public_updated_at) { research_output['public_updated_at'] }
 
   before do
+    allow_any_instance_of(DocumentPolicy).to receive(:departmental_editor).and_return(true)
```

Comment:
Would it make sense to stub `Rails.env.development?` instead? That would mean a bit more of the code runs and it'd also add a bit of confidence that the DFID finder appears in development.